starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,59 @@
 // Constructor
-// function Chronometer() {
 
-// }
 
-// Chronometer.prototype.startClick = function () {
+function Chronometer() {
+    this.currentTime = 0,
+    this.intervalId = 0
+}
 
-// };
+Chronometer.prototype.startClick = function (setTime) {
+    
+        var that = this
+        setInterval (function(){
+           var time = that.currentTime ++;
+            that.setTime();
+        },1000)
 
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
+    };
 
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
+Chronometer.prototype.setMinutes = function () {
+    return parseInt(this.currentTime/60);
+ };
 
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
+Chronometer.prototype.setSeconds = function () {
+  return Math.round(this.currentTime - 60 * this.setMinutes())
+ };
 
-// Chronometer.prototype.setTime = function () {
+Chronometer.prototype.twoDigitsNumber = function (zegar) {
+  if (zegar < 10){
+      return 0 + zegar.toString()
+  }else{
+      return zegar.toString()
+  }
+};
 
-// };
+ Chronometer.prototype.setTime = function () {
```

Comment:
setMinutes y setSeconds son MÉTODOS. Normal que no os funcione. Si véis que algo no funciona como debería, keep calm and console.log, era relativamente fácil de debuggar.